setup.py

```diff
@@ -38,11 +35,6 @@ def run(self):
         _sdist.run(self)
 
 
-class install(_install):
```

Comment:
Yes, I saw your comment, I'm afraid I've only just picked it up, sorry about that.

I didn't realise that my PR broke the pypi packaging, so I can immediately understand why you removed it.

What you suggest does make sense :)